internal/datastore/crdb/migrations/zz_migration.0008_add_transaction_metadata_table.go

```diff
@@ -0,0 +1,75 @@
+package migrations
+
+import (
+	"context"
+	"strings"
+
+	"github.com/jackc/pgx/v5"
+)
+
+const (
+	// ttl_expiration_expression support was added in CRDB v22.2, but the E2E tests
+	// use v21.2.
```

Comment:
Can you clarify what's the difference between the TTL support in 22.2 vs 21.2?

Reply:
The `ttl_expiration_expression` property was added in 22.2

Reply:
yeah but the pre 22.2 version has a different expression, what does it do?

Reply:
It just expires after 1 day, vs using the column's value